src/App.js

```diff
@@ -24,14 +22,17 @@ function App() {
   };
 
   return (
-
     <BrowserRouter>
       <Header isAddSelected={isAddSelected} onListSelect={selectListHandler} />
       <div className="App">
-       <Items />
+        {/* <Items /> */}
```

Comment:
Feel free to completely remove this if you don't need it anymore. Comments are harmless to the user, but comments are valuable for other developers. One day someone will wonder why this is here, and it may cause confusion 😄

Reply:
I had a similar question since it is duplicated? Agreed, can remove!

Reply:
Will do. I think it may have just been a carry over that I was too scared to remove completely. Haha. But I think you're right, it doesn't look like we need it for now.